app/api/paragraphExtraction/domain/PXExtractorsQueryService.ts

```diff
@@ -7,6 +7,8 @@ type GetExtractorsOutput = {
   _id: string;
   sourceTemplateId: string;
   targetTemplateId: string;
+  paragraphNumberPropertyId: string;
```

Comment:
@RafaPolit we needed this in the client, is it Ok that we are extending the returned values of this service endpoint?